src/utils/fetchAPI.tsx

```diff
@@ -7,3 +7,12 @@ export default async function fetchAPI(url: string) {
   const body = await response.json();
   return body;
 }
+
+export function deleteEntry(url: string, id: number) {
```

Comment:
Maybe you want to rename the file to `api.tsx` now because it doesn't only include the fetch function now? :)

Reply:
Renamed it